CONTRIBUTING.md

```diff
@@ -20,16 +20,18 @@ Please contribute links to packages/projects you have used or are familiar with.
 
 ## Quality standard
 
-To stay on the list, package repositories should adhere to these quality standards:
+To be on the list, project repositories should adhere to these quality standards:
 
-- Generally useful to the community
-- Functional
-- Actively maintained (even if that just means acknowledging open issues when they arise)
-- Stable, or progressing toward stable
-- Documented (preferably godoc.org)
-- Tests are preferred (when possible)
+- Code is functional
+- Generally useful to the wider community of Go programmers
+- Actively maintained
+  - Regular, recent commits
+  - Or, for finished projects, issues and pull requests are responded to
+- Stable or progressing toward stable
+- Thoroughly documented (README, godoc comments, etc.)
+- Tests, where practical
```

Comment:
I like that idea, although I do have some reservations about golint. It's a great tool but the output is mainly suggestions... for example, I disagree with it on several points of style.

What about goreportcard.com - it does use golint but it also incorporates other factors into the grade...